network/ovs_endpointclient_linux.go

```diff
@@ -109,18 +115,10 @@ func (client *OVSEndpointClient) AddEndpointRules(epInfo *EndpointInfo) error {
 	}
 
 	log.Printf("[ovs] Get ovs port for interface %v.", client.hostVethName)
-	containerOVSPort, err := client.ovsctlClient.GetOVSPortNumber(client.hostVethName)
-	if err != nil {
-		log.Printf("[ovs] Get ofport failed with error %v", err)
-		return err
-	}
+	containerOVSPort := epInfo.VethOVSSwitchPort
 
 	log.Printf("[ovs] Get ovs port for interface %v.", client.hostPrimaryIfName)
-	hostPort, err := client.ovsctlClient.GetOVSPortNumber(client.hostPrimaryIfName)
-	if err != nil {
-		log.Printf("[ovs] Get ofport failed with error %v", err)
-		return err
-	}
+	hostPort := epInfo.VethOVSSwitchPort
```

Comment:
this should be extIf.OvsSwitchPort

Reply:
this function does not have access to extIf? epInfo is the struct passed in here so I thought to fetch from epInfo

Reply:
replied below